### PR TITLE
update controller broker glossterm

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2443
Review deadline: Thursday, May 16

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [X] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/terms/partials/controller-broker.adoc [hunk 1]
```diff
@@ -1,12 +1,4 @@
 === controller broker
 :term-name: controller broker
-:hover-text: A broker that’s responsible for managing operational metadata for a Redpanda cluster. At any given time, one active controller exists in a cluster. 
+:hover-text: A broker that manages operational metadata for a Redpanda cluster and ensures replicas are distributed among brokers. At any given time, one active controller exists in a cluster. If the controller fails, another broker is automatically elected as the controller.
```
Comment (Deflaimun) [inline modules/terms/partials/controller-broker.adoc:3]:
```suggestion
:hover-text: A broker that manages operational metadata for a Redpanda cluster and ensures replicas are distributed among brokers. At any given time, one active controller exists in a cluster. If the controller fails, another broker is automatically elected as the controller. 
```

File: modules/terms/partials/controller-broker.adoc [hunk 2]
```diff
@@ -1,12 +1,4 @@
...
```
Comment (Deflaimun) [inline modules/terms/partials/controller-broker.adoc]:
```suggestion
:hover-text: A broker that manages operational metadata for a Redpanda cluster and ensures replicas are distributed among brokers. At any given time, one active controller exists in a cluster. If the controller fails, another broker is automatically elected as the controller.
```

**Comment by Deflaimun** [comment]
Force preview

**Review by ztlpn — APPROVED** [review]

**Review by Feediver1 — APPROVED** [review]
LGTM

**Pull request merged**